packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -308,7 +308,7 @@ export default {
             'getGeoLocation'
         ]),
 
-        ...mapActions('analytics', [
+        ...mapActions('checkoutAnalytics', [
```

Comment:
Would be good to centralise this name so we can change it once throughout. E.g [here](https://github.com/justeat/fozzie-components/blob/master/packages/components/molecules/f-searchbox/src/services/constants.js#L13)

Reply:
@kevinrodrigues I've changed `checkout` to `fCheckoutModule` and `analytics` to `fCheckoutAnalyticsModule` Are these good?